src/app/views/censusdata/create.html

```diff
@@ -94,73 +73,17 @@
-                    <tbody>
-                    <tr>
-                        <td><input type="text" class="form-control" placeholder="Species Name"></td>
-                        <td><input type="text" class="form-control" placeholder="No. of Adults"></td>
-                        <td><input type="text" class="form-control" placeholder="No. of Juvenile"></td>
-                        <td><input type="text" class="form-control" placeholder="Primary Activity"></td>
-                        <td><input type="text" class="form-control" placeholder="Secondary Activity"></td>
-                        <td><select id="habitatData" class="form-control" >
-                            <option ng-repeat="habitat in habitatInfo" value="{{habitat.Id}}">{{habitat.Description}}</option>
-                        </select></td>
-                        <td><select id="feedingSuccess" class="form-control" >
-                        <option ng-repeat="feed in feedingRateInfo" value="{{feed.Id}}">{{feed.Description}}</option>                  
-                        </select></td>
-                        <td><span class="glyphicon glyphicon-edit"></span></td>
-                        <td><span class="glyphicon glyphicon-trash"></span></td>
-                    </tr>        
-                    <tr>
-                        <td colspan="9">
-                        <button type="submit" class="btn btn-primary">
-                            <i class="glyphicon glyphicon-plus"></i> Add More
-                        </button>
-                        </td>
-                    </tr>             
                     
-                    </tbody>
-                </table>
-        <!--             
-                    </div>
-                </div>-->
-                </div>    
-            </div>  
-            <div class="form-group">
-                <label class="control-label col-md-3" for="comments">Comments:</label>
-                <div class="col-md-9">
-                <textarea type="text" class="form-control" rows="5" id="comments" placeholder="Enter Comments" ng-model="censusForm.comments"></textarea>
-                </div>    
-            </div>          
             </div>    
             <div class="panel-footer">
-            <div class="form-group">
-                <input type="button" class="btn btn-primary col-md-offset-1 col-md-2" value="&laquo; Step Back" disabled >           
-                <div class="col-md-offset-4 col-md-5">          
-                <input type="submit" class="btn btn-primary" value="Save & Exit">
-                <input type="button" class="btn btn-primary" value="Proceed Further &raquo;">
+            <div class="form-group">          
+                <div class="col-md-offset-8 col-md-4">          
+                    <input type="submit" class="btn btn-primary" value="Save & Exit">
+                    <a role="button" class="btn btn-primary" href="#/censusdata/create2">Proceed Further &raquo;</a>
```

Comment:
Rather than "Proceed Further", let's put "Next", which is more common and will be recognized more quickly.